doc/en/multitest.rst

```diff
@@ -173,20 +173,23 @@ Name customization
 ------------------
 
 Testplan supports customizing test suite name or testcase name, which will be saved
-in test report and displayed on UI. displayed inUse ``name`` argument in @testcase
-or ``custom_name`` argument in @testsuite, while ``custom_name`` can be a normal string
-or a callable. Some examples:
+in test report and displayed on UI. Argument ``name`` is available in @testcase and
+should be a normal string, while in @testsuite it can be a normal string or a callable.
+To customize names for parametrized testcases another argument ``name_func`` is useful,
+refer to the document of :ref:`name_func <parametrization_custom_name_func>`.
+Some examples:
 
-    * ``@testcase(name="Testcase name")``
-    * ``@testsuite(custom_name="Test suite name")``
-    * ``@testsuite(custom_name=lambda self, original_name: "{}_{}".format(original_name , id(self)))``
+    * @testcase(name="My Testcase")
+    * @testcase(name="My Testcase", name_func=lambda func_name, kwargs: func_name.upper())
```

Comment:
Usage of name_func without parameterization is improper?

Reply:
removed